source/rust_verify/src/singular.rs

```diff
@@ -5,6 +5,7 @@ use air::singular_manager::SingularManager;
 use sise::Node;
```

Comment:
For all the error messages in this file, and `well_formed` can you clarify whether a certain expression or feature is unsupported. For example:

`usage of the keyword {} is not supported when using the integer_ring solver`
instead of
"usage of reserved keyword at variable name: {} in integer_ring",

Reply:
I made some adjustments to the error message.

Reply:
👍